lib/index.ts

```diff
@@ -174,14 +174,12 @@ export function createAdapter(
           },
         },
       ],
-      {
-        resumeAfter: resumeToken,
-      }
+      changeStreamOpts
```

Comment:
Actually, I don't think we need to store the `resumeToken` here, we should be able to refer to `changeStream.resumeToken` directly:

```js
const changeStreamOpts = {};

if (changeStream && changeStream.resumeToken) {
  changeStreamOpts.resumeAfter = changeStream.resumeToken;
}

changeStream = mongoCollection.watch(...);
```

What do you think?